skiko/src/commonMain/kotlin/org/jetbrains/skiko/Annotations.kt

```diff
@@ -27,3 +27,12 @@ annotation class DelicateSkikoApi
             "and can have bugs."
 )
 annotation class ExperimentalSkikoApi
+
+/**
+ * Marks declarations that we did not intend to make public.
+ */
+@RequiresOptIn(
+    level = RequiresOptIn.Level.WARNING,
+    message = "This is an internal API. Please don't use it."
+)
+annotation class SkikoInternalApi
```

Comment:
Why `ExperimentalSkikoApi` But `SkikoInternalApi`? I'd suggest `InternalSkikoApi` instead.

PS Please do not merge changes that changes some public things so fast

Reply:
Oh, it's not to master. I see